lib/extra/swagger-shim.ts

```diff
@@ -164,3 +164,6 @@ export function PartialType() {
 export function PickType() {
   return () => {};
 }
+export function getSchemaPath() {
+  return () => {}
```

Comment:
`getSchemaPath` returns a string, so I guess this should be
```suggestion
  return ''
```

https://github.com/nestjs/swagger/blob/ae525047c27c4c0c4383b4db8ab6f433a510be4b/lib/utils/get-schema-path.util.ts#L3-L6

Reply:
oh yes this was brain death bad copy paste